types/react-albus/react-albus-tests.tsx

```diff
@@ -37,15 +37,20 @@ const Example = () => (
                         </div>
                     )}
                 />
-                <Step
-                    id="dumbledore"
-                    render={({ previous }) => (
```

Comment:
Instead of replacing the render test, can you add a test that confirms you can use `children`? 
That way we're sure people can use  both `render` and `children` props

Reply:
If you look at the whole file you will see that all of the use-cases are covered.
- `merlin` = `render`
- `gandalf` = also `render`, but with context
- `dumbledore` = function-as-children
- `harry` = plain `children` without context

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/d23748c6c40ca77013d5c33475ae5aecf804d37a/types/react-albus/react-albus-tests.tsx

Is this sufficient or should I still revert my edit and add a seperate test?

Reply:
My bad!